docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -116,22 +116,63 @@ Teleport will always look at the oldest version of a recording.
 
-There are two supported options when using AWS:
-- Use `cert-manager` to provision and automatically renew ACME certs (described in [Step 4a](#step-4a7-install-and-configure-cert-manager-to-handle-tls)).
-  - This approach is recommended if you require CLI access to web applications using client certificates via Teleport Application Access.
-- Use AWS Certificate Manager to handle TLS termination with AWS-managed certificates (described in [Step 4b](#step-4b7-configure-teleport-to-use-acm-to-handle-tls)).
-  - This will prevent Teleport Application Access from working via CLI using client certificates. Application Access will still work via a browser.
+### Determining an approach
 
-You must choose only one of these options.
+There are three supported options when using AWS. You must choose only one of
+these options:
 
+#### Using `cert-manager` 
 
-## Step 4a/7. Install and configure cert-manager to handle TLS
+You can use `cert-manager` to provision and automatically renew TLS credentials
+by completing ACME challenges via Let's Encrypt. We recommend this approach if
+you require CLI access to web applications using client certificates via
+Teleport Application Access.
 
-In this example, we are using multiple pods to create a High Availability Teleport cluster. As such, we will be using
-`cert-manager` to centrally provision TLS certificates using Let's Encrypt. These certificates will be mounted into each
-Teleport pod, and automatically renewed and kept up to date by `cert-manager`.
+#### Using AWS Certificate Manager
+
+You can use AWS Certificate Manager to handle TLS termination with AWS-managed
+certificates.
+
+You should be aware of the limitations for using AWS Certificate Manager to
+provision TLS credentials for Teleport:
+
+- This will prevent Teleport Application Access from working via CLI using
+  client certificates. Application Access will still work via a browser.
+- Command-line Application Access does not work with ACM. Using ACM will prevent
+  Teleport from facilitating Application Access via CLI (using client
+  certificates), as Teleport will not be handling its own TLS termination.
+- Using ACM through a AWS Load Balancer prevents the required traffic for
+  Postgres or MongoDB through Teleport's web port. If you choose to use the ACM
+  approach, we will show you how to configure a separate listener for Postgres
+  or MongoDB. 
+
+#### Using your own TLS credentials
+
+With this approach, you are responsible for determining how to obtain a TLS
+certficiate and private key for your Teleport cluster, and for renewing your
+credentials periodically. Use this approach if you would like to use a trusted
+internal certificate authority instead of Let's Encrypt or AWS Certificate
+Manager.
+
+<Notice type="warning">
+
+If you would like the Teleport Application Service and Database Service to
+function as expected, you should use the `cert-manager` approach unless there is
+a specific reason to use ACM.
+
+</Notice>
```

Comment:
Does this notice belong in the previous section?

Reply:
It does 🤦

Reply:
https://www.youtube.com/watch?v=Ag1o3koTLWM